frontend/src/pages/AddWorkoutPage.tsx

```diff
@@ -33,6 +35,7 @@ export default function AddWorkoutPage() {
             await axios.post('/api/workouts', formData);
             setFormData({ name: '', description: '' });
             alert(`Thanks. Workout "${formData.name}" added.`);
+            navigate("/");
```

Comment:
no navigate here. after adding a workout, we should stay on the adding-page. only after edit, we want to go to the details page